vpr/src/base/region.cpp

```diff
@@ -1,7 +1,7 @@
 #include "region.h"
 
 /// @brief sentinel value for indicating that a subtile has not been specified
-constexpr int NO_SUBTILE = -1;
```

Comment:
change back to -1, if the subtile is not specified, need to make logic in init place understand that that means any subtile should work

Reply:
and in move generators, placement checkers. Have a method that checks if no subtile - returns if subtile constrained, and then in code can just check if subtile is constrained

Reply:
potentially, could have a rule that each partition either has subtiles specified or doesn't, but if don't need this rule don't have to add it, just see as coding